src/app/Main/Body/QueryPage/QueryPageContent/Parameter/createParameterList.service.tsx

```diff
@@ -0,0 +1,64 @@
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+import { TFunction } from "i18next";
+import { ListItem, ListItemText } from "@mui/material";
+import { FilterType } from "../../../../../Shared/Model/Filter.model";
+import { getMicroorganismLabelService } from "../../Services/getMicroorganismLabel";
+import { replaceAll } from "../../../../../Core/replaceAll.service";
+
+const spaceStyle = css`
+    padding: 0;
+    margin: 0;
+`;
+const parameterValue = css`
+    margin-top: 0;
+    margin-left: 2em;
+    span {
+        letter-spacing: 0;
+    }
+`;
+
+export function createParameterListService(

```

Comment:
Reject: Please do not use component factories like this service. This is not necessary in your app and blurs the distinction between components and services. Styles and html code should be only used in conventional react components. So move this back to a component.
You could use a mapper services for extracting the translation stuff and create the data required by the parameterListComponent.

Reply:
Done. But not sure if I solved it exactly as you imagined.

Reply:
Not exactly, but then you have to refactor the microorgaLabelService too (i missed that). But the reject is done.